crypto/dso/dso_win32.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include "dso_locl.h"
+#include "e_os.h"
```

Comment:
On *side note* there is ambiguity here. Remember I mentioned that winsock[2].h is position-sensitive? And since e_os.h includes it, it, e_os.h, becomes position-sensitive too. In more practical terms in order to place e_os.h in specific position, you have to be sure that none of earlier headers include <windows.h>. Fool-proof way is to always include e_os.h first. Yes, you can observe multiple cases when e_os.h is not first, but this only means that it happens to work out in that particular cases. Besides, "position sensitivity" is rather about what gets declared, not compilation failures, and in those cases where e_os.h is not first (or early enough) you're off the hook if you don't need those missing declarations. But arguably one can wonder if it still makes sense to formulate some kind of rule, such as "e_os.h always first", anyway. You know, because it's easier to apply simple rule, than to figure out if position is right on per-case basis... Once again, this was a side note...